src/serac/infrastructure/output.cpp

```diff
@@ -18,8 +18,39 @@
 
 namespace serac::output {
 
+namespace detail {
+std::string file_format_string(const FileFormat file_format)
+{
+  std::string value = "";
+  if (file_format == FileFormat::JSON) {
+    value = "json";
+  } else if (file_format == FileFormat::HDF5) {
+    value = "hdf5";
+  } else if (file_format == FileFormat::YAML) {
+    value = "yaml";
+  }
+  return value;
+}
+}  // namespace detail
+
+void outputSummary(const axom::sidre::DataStore& datastore, const std::string& data_collection_name,
+                   const FileFormat file_format)
+{
+  auto [_, rank] = getMPIInfo();
+  if (rank != 0) {
+    return;
+  }
+
+  // FIXME: remove this error when implemented, do we even want hdf5 curves?
```

Comment:
We don't want HDF5 curves.

Reply:
it makes even less sense with the other summary info.